wolfssl/wolfcrypt/settings.h

```diff
@@ -228,6 +228,10 @@
     #include "wolfSSL.wolfSSL_conf.h"
 #endif
 
+#ifdef HAVE_WC_INTROSPECTION
+WOLFSSL_API const char *libwolfssl_configure_args(void);
```

Comment:
settings.h shouldn't be declaring functions.  Is logging.h more appropriate?

Also, wolfSSL functions should begin with wolfSSL_

Reply:
Done and done.